mkdocs.yml

```
@@ -1156,6 +1156,7 @@ nav:
         - Install private nodes: integrations/creating-nodes/deploy/install-private-nodes.md
   - Hosting n8n:
     - hosting/index.md
+    - Community edition: hosting/community-edition-features.md
```

```suggestion
  - Hosting n8n:
    - Overview: hosting/index.md
    - Community vs Enterprise: hosting/community-edition-features.md
```
1. I think the new page is more about the comparison between the community and enterprise edition rather than what community edition is. We mention what the community edition is missing, which is more or less a list of all the enterprise features. So how about we rename it to 'Community vs Enterprise' or something along those lines?
2. I personally think 'Hosting n8n' looks more like a section header than an overview page of its own, and if we add a new page underneath it, maybe users won't think to click it? -> Instead, what if we make it into an 'Overview' page instead?

---

I'll take it. Thank you.